docker-compose.yml

```diff
@@ -1,6 +1,15 @@
 version: "3.1"
 
 services:
+  # Bitcoin node for development, if you want to start your own node, 
+  # uncomment below lines:
+  # bitcoin:
```

Comment:
Let's name it `bitcoin-node` and also I'm not sure if that should be commented out. I think you did that mostly, because someone can accidentally run node, while typing `docker-compose up` correct?

Reply:
exactly, I'm wondering if it should be a part of the docker-compose, maybe I should create a wiki page about local Bitcoin development and move this code there?

Reply:
or, move that to completely new docker-compose file. Like `docker-compose.local.yml` and put it there. I believe to execute that, you need to run `docker-compose -f <filename>`. @ivanopagano @piotrkosecki what do you think about that?

Reply:
this is the `local` docker-compose file, as the one located in the`./docker` directory has full integration for the current conseil/lorre/postgres. I would vote to keep it here. Just change the wiki to mention current development containers.

Reply:
I agree with @piotrkosecki , every dev can comment out whatever is needed.

Personally I'd comment out the less needed components, like graphql and bitcoin, and commit it like that.
For me it's the most ergonomic solution for out-of-the-box development. YMMV